_example/complete/windows_web_app/versions.tf

```diff
@@ -0,0 +1,12 @@
+terraform {
+  required_version = ">= 1.9.0"
+}
+
+terraform {
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "=3.109.0" #3.100.0
```

Comment:
Remove `#3.100.0` and make it `>= 3.109.0`.

Reply:
Imp;iment all the changes specified